frontend/views/containers/proposals/ProposalItem.vue

```diff
@@ -326,7 +326,7 @@ export default ({
         !this.currentGroupState._vm?.authorizedKeys?.[inviteKeyId] ||
         // If _notAfterHeight is *not* undefined, it means that the key has been
         // revoked. Hence, it cannot be used
-        this.currentGroupState._vm.authorizedKeys[inviteKeyId]._notAfterHeight !== undefined ||
```

Comment:
Are you certain about this change? Is this something @corrideat missed?

Reply:
I think this was a fix for a failing cypress test at that time which is unrelated to this work. If that has been fixed in the `master`, I will put it back to whatever it is there.

Reply:
changed it according to `master`